ui/tract-renderer/src/main/java/org/cru/godtools/tract/widget/PageContentLayout.java

```diff
@@ -736,4 +824,28 @@ public LayoutParams(@NonNull final LayoutParams p) {
             this.childType = p.childType;
         }
     }
+
+    //region bounce thread calls
+    /**
+     *  This runnable object is used to create a loop that will run the animation.
+     */
+    private Runnable bounceAnimationRunnable = new Runnable() {
```

Comment:
I would need to check, but this `Runnable` might leak in the `Handler` if the `PageContentLayout` is destroyed.

Reply:
https://www.androiddesignpatterns.com/2013/01/inner-class-handler-memory-leak.html